tests/ansible_collections/roles/install-testing-deps/tasks/main.yml

```diff
@@ -9,9 +9,13 @@
   ignore_errors: True
   register: pip_command
 
-- name: Install pip if not present
+- name: Install pip on centos7/oracle7 if not present
```

Comment:
what about to use "rhel 7 like system" instead of hardcoded names?

Reply:
Changed it to according to your suggestion.